pkg/kubectl/resource_printer.go

```diff
@@ -504,6 +504,10 @@ func translateTimestamp(timestamp util.Time) string {
 }
 
 func printPod(pod *api.Pod, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
+	return printPodBase(pod, w, withNamespace, wide, showAll, true, columnLabels)
+}
+
+func printPodBase(pod *api.Pod, w io.Writer, withNamespace bool, wide bool, showAll bool, showIfTerminating bool, columnLabels []string) error {
```

Comment:
Yes, end user indicate it, the command option is "-a, --show-all[=false]: When printing, show all resources (default hide terminated pods.)"  When a pod named test is terminated, "kubectl get pods test" doesn't show anything, "kubectl get pods test -a" could show it. But I don't think "kubectl get pods test -a" is easy to use. I think "kubectl get pods test" should show it.